File: 2639-separate-the-digits-in-an-array/separate-the-digits-in-an-array.java
class Solution{
    public int[] separateDigits(int[] nums){
        int n=nums.length;

        StringBuilder str=new StringBuilder();
        for(int i=0; i<n; i++){
            str.append(nums[i]);
        } 

        int[] res=new int[str.length()];
        for(int i=0; i<str.length(); i++){
            res[i]=str.charAt(i)-'0';
        }

        return res;
    }
}

File: Easy/2639-separate-the-digits-in-an-array/separate-the-digits-in-an-array.cpp
class Solution{
public:
    vector<int> separateDigits(vector<int> &nums){
        int n=nums.size();

        string str="";
        for(int i=0; i<n; i++){
            str += to_string(nums[i]);
        } 

        vector<int> res;
        for(auto it: str){
            res.push_back(it-'0');
        }

        return res;

    }
};